ssm/agents.py

```diff
@@ -379,3 +383,30 @@ def get_dns(dn_file, log):
 
     log.debug('%s DNs found.', len(dns))
     return dns
+
+
+def get_banned_dns(log):
+    """Retrieve the list of banned dns"""
+    banned_dns_list = []
```

Comment:
For a variable that's clearly created as a list and only used in this local scope, I'd say drop the `_list` part from the name.

Reply:
done